### DISCUSSION TITLE
4.26.0 introduces breaking changes for some 3rd party packages

### DISCUSSION THREAD

**Discussion by Blindmikey** [laravel/nova-issues#5650] [Q&A]
- Laravel Version: 10.14.0
- Nova Version: 4.26.0
- PHP Version: 8.1

### Description:
Perhaps this is up to 3rd party package maintainers but I figured I post this here as well. 

Breaks some well-used 3rd party packages:
https://github.com/outl1ne/nova-settings/issues/166
https://github.com/whitecube/nova-flexible-content/issues/464
Possibly more. 

Causes packages to throw an error concerning `::forceFill()`
such as:
`Call to undefined method stdClass::forceFill()`
`Call to undefined method Whitecube\NovaFlexibleContent\Layouts\Layout::forceFill()`

### Detailed steps to reproduce the issue on a fresh Nova installation:
Step-by-step to reproduce on a clean install found here: https://github.com/outl1ne/nova-settings/issues/166#issuecomment-1610784910

**Comment by crynobone** [comment]
Convert `\stdClass` to `\Laravel\Nova\Support\Fluent`
